internal/logsink/logsink.go

```diff
@@ -236,7 +238,9 @@ func textPrintf(m *Meta, textSinks []Text, format string, args ...any) (n int, e
 		var tmp [19]byte
 		buf.Write(strconv.AppendInt(tmp[:0], int64(m.Line), 10))
 	}
-	buf.WriteString("] ")
+	buf.WriteString(" (")
+	buf.WriteString(fmt.Sprintf("%v",m.Thread))
```

Comment:
The fmt.Sprintf with `%v` is expensive - can we put this back to the original faster way where they cast to uint64 ?

```
nDigits(buf, 7, uint64(m.Thread), ' ')
```